provider/resource_keycloak_ldap_user_federation_test.go

```diff
@@ -128,6 +129,15 @@ func TestAccKeycloakLdapUserFederation_basicUpdateAll(t *testing.T) {
 	firstValidatePasswordPolicy := randomBool()
 	firstPagination := randomBool()
 
+	var firstTrustEmail, secondTrustEmail bool
+	if keycloakMajorVersion(t) >= 8 {
+		firstTrustEmail = randomBool()
+		secondTrustEmail = !firstTrustEmail
+	} else {
+		firstTrustEmail = false
+		secondTrustEmail = false
```

Comment:
Looks like this is still failing.

It might be best to remove the `trustEmail` stuff from this test and make a separate test specifically for testing this attribute.  Then we can skip that test if the version of Keycloak doesn't support it.